plugin.json

```diff
@@ -15,27 +15,27 @@
         "footer": "",
         "settings": [{
             "key": "TeamChannel",
-            "display_name": "Team/Channel",
+            "display_name": "Channel to send notifications to:",
             "type": "text",
-            "help_text": "Name of the Team and Channel you want to create/use to receive the SNS messages. Split by comma. ie. teamA,channelXYZ"
+            "help_text": "The channel to send notifications to, specified in the format `teamname,channelname`. If the specified channel does not exist, the plugin will create the channel for you.\n\nNote: Must be the team and channel handle used in the URL. For example, in the following URL, set the value to `myteam,mychannel`: https://example.com/myteam/channels/mychannel"
         },
         {
             "key": "AllowedUserIds",
-            "display_name": "AllowedUserIds",
+            "display_name": "Authorized users who can accept AWS SNS subscriptions:",
             "type": "text",
-            "help_text": "Users ID authorized to accept the subscription."
+            "help_text": "List of users authorized to accept AWS SNS subscriptions to a Mattermost channel. Must be a comma-separated list of user ID's.\n\nTip: Use the [mattermost user search](https://mattermost.com/pl/cli-mattermost-user-search) CLI command to determine a user ID."
```

Comment:
Created a task for Marketing to redirect https://mattermost.com/pl/cli-mattermost-user-search to https://docs.mattermost.com/administration/command-line-tools.html#mattermost-user-search

Until done (should be within a few days), this link takes user to a 404 error page.